api/client/utils.py

```diff
@@ -0,0 +1,99 @@
+"""Miscellaneous utility functions for operations not specific to the API client."""
```

Comment:
is this intentionally committed? not seeing any callers

Reply:
Whoops. Stuff got added back in when resolving a merge conflict. Removed. Now it's importing from the utils file